"Create term" test fails with WP minimum 5.7

### Describe the bug

```
 1) Command: createTerm
       Should be able to Create a category:
     AssertionError: Timed out retrying after 4000ms: expected '[ <div.update-nag.notice.notice-warning.inline>, 1 more... ]' to contain 'Category added'
      at Context.eval (webpack:///./e2e/create-term.test.js:18:26)
```

### Steps to Reproduce

https://github.com/10up/cypress-wp-utils/actions/runs/3668615715/jobs/6202100398#step:7:604

### Screenshots, screen recording, code snippet

_No response_

### Environment information

_No response_

### WordPress information

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/tests/cypress/e2e/create-term.test.js
+++ b/tests/cypress/e2e/create-term.test.js
@@ -13,33 +13,18 @@ describe('Command: createTerm', () => {
   it('Should be able to Create a category', () => {
     const termName = 'Category ' + randomName();
     cy.createTerm(termName);
-    cy.get('body').then($body => {
-      if ($body.find('.notice').is(':visible')) {
-        cy.get('.notice').should('contain', 'Category added');
-      }
-    });
     cy.get(`.row-title:contains("${termName}")`).should('exist');
   });
 
   it('Should be able to Create a tag', () => {
     const termName = 'Tag ' + randomName();
     cy.createTerm(termName, 'post_tag');
-    cy.get('body').then($body => {
-      if ($body.find('.notice').is(':visible')) {
-        cy.get('.notice').should('contain', 'Tag added');
-      }
-    });
     cy.get(`.row-title:contains("${termName}")`).should('exist');
   });
 
   it('Duplicate category should not be created', () => {
     const termName = 'Category ' + randomName();
     cy.createTerm(termName);
-    cy.get('body').then($body => {
-      if ($body.find('.notice').is(':visible')) {
-        cy.get('.notice').should('contain', 'Category added');
-      }
-    });
     cy.get(`.row-title:contains("${termName}")`).should('exist');
 
     cy.createTerm(termName);
@@ -52,11 +37,6 @@ describe('Command: createTerm', () => {
   it('Duplicate tag should not be created', () => {
     const termName = 'Tag ' + randomName();
     cy.createTerm(termName, 'post_tag');
-    cy.get('body').then($body => {
-      if ($body.find('.notice').is(':visible')) {
-        cy.get('.notice').should('contain', 'Tag added');
-      }
-    });
     cy.get(`.row-title:contains("${termName}")`).should('exist');
 
     cy.createTerm(termName, 'post_tag');
@@ -79,12 +59,6 @@ describe('Command: createTerm', () => {
     });
 
     // Assertions for parent category
-    cy.get('body').then($body => {
-      if ($body.find('.notice').is(':visible')) {
-        cy.get('.notice').should('contain', 'Category added');
-      }
-    });
-
     cy.get(`.row-title:contains("${parentCategory.name}")`).then(
       $parentLink => {
         // Assertions of parent category

